transport/connection.go

```diff
@@ -281,12 +278,24 @@ func (t *gettyTCPConn) send(pkg interface{}) (int, error) {
 			t.wLastDeadline = currentTime
 		}
 	}
+	if buffers, ok := pkg.([][]byte); ok {
+		netBuf := net.Buffers(buffers)
+		if length, err := netBuf.WriteTo(t.conn); err == nil {
+			atomic.AddUint32(&t.writeBytes, (uint32)(length))
+			atomic.AddUint32(&t.writePkgNum, (uint32)(len(buffers)))
+		}
+		log.Debug("now:%s, length:%d, err:%s", currentTime, length, err)
```

Comment:
I think u should delete this log now. If it is useful, u should add more info like url etc.